components/Padrinho/Padrinho.module.scss

```diff
@@ -0,0 +1,13 @@
+.ctaWrapper {
```

Comment:
You don't need a new CSS wrapper for it. Use the same pattern of [slug] page

Reply:
I'm afraid I'm not following you. Wasn't I supposed to create the Padrinho component, though?